packages/common-all/src/utils.ts

```diff
@@ -865,3 +865,19 @@ export function cleanName(name: string): string {
 export function normalizeUnixPath(fsPath: string): string {
   return path.posix.format(path.parse(fsPath));
 }
+
+/** Wrapper(s) for easier testing, to wrap functions where we don't want to mock the global function. */
+export class Wrap {
```

Comment:
can you add documentation for this in [[Cook|dendron://dendron.docs/dev.process.qa.test#cook]]?

feel free to merge afterward (and add to weekly)